examples/mongodb/README.md

```diff
@@ -121,65 +132,81 @@ To restore the missing data, you should use the backup that you created before.
 As a part of restore operation in MongoDB ReplicaSet, we are deleting data from the Secondary replicas to allow MongoDB to use `Initial Sync` for updating Secondaries as documented [here](https://docs.mongodb.com/manual/tutorial/restore-replica-set-from-backup/#update-secondaries-using-initial-sync)
 
 ```bash
-$ kanctl --namespace kasten-io create actionset --action restore --from "backup-llfb8"
+$ kanctl --namespace kanister create actionset --action restore --from "backup-llfb8"
 actionset restore-backup-llfb8-64gqm created
 
 # View the status of the ActionSet
-kubectl --namespace kasten-io describe actionset restore-backup-llfb8-64gqm
+kubectl --namespace kanister get actionset restore-backup-llfb8-64gqm
+NAME                         PROGRESS   LAST TRANSITION TIME   STATE
+restore-backup-llfb8-64gqm   100.00     2022-12-09T10:06:39Z   complete
 ```
 
 You should now see that the data has been successfully restored to MongoDB!
 
 ```bash
-$ mongo admin --authenticationDatabase admin -u root -p $MONGODB_ROOT_PASSWORD --quiet --eval "db.restaurants.find()"
-{ "_id" : ObjectId("5d778f987799be09da2f8ade"), "name" : "Roys", "cuisine" : "Hawaiian", "id" : "8675309" }
+# Connect to MongoDB primary pod
+$ kubectl exec -ti my-release-mongodb-0 -n mongo-test -- bash
+
+$ mongosh admin --authenticationDatabase admin -u root -p $MONGODB_ROOT_PASSWORD --quiet --eval "db.restaurants.find()"
+[
+  {
+    _id: ObjectId("6393059b7844b3c1445e9d14"),
+    name: 'Roys',
+    cuisine: 'Hawaiian',
+    id: '8675309'
+  }
+]
 ```
 
 ### Delete the Artifacts
 
 The artifacts created by the backup action can be cleaned up using the following command:
 
 ```bash
-$ kanctl --namespace kasten-io create actionset --action delete --from "backup-llfb8"
+$ kanctl --namespace kanister create actionset --action delete --from "backup-llfb8" --namespacetargets kanister
```

Comment:
```suggestion
$ kanctl create actionset --action delete --from "backup-llfb8" --namespacetargets kanister
```

Reply:
LGTM apart from above comment.

Reply:
Don't we need `--namespace` flag for the ActionSet to be created in the `kanister` namespace? `--namespacetargets` only sets it as a subject right?

Reply:
namespace is required here. otherwise it will create actionset in default namespace

Reply:
ok, makes sense. Thank you for explaining.